rskj-core/src/main/java/co/rsk/config/RskSystemProperties.java

```diff
@@ -417,4 +417,8 @@ public int getPeerP2PPingInterval(){
     public Integer getGasPriceBump() {
         return configFromFiles.getInt("transaction.gasPriceBump");
     }
+
+    public int getStatesCacheSize() {
+        return configFromFiles.getInt("cache-size.states");
```

Comment:
Does this mean the config var is "states" in config section "cache-size" ? I seems that the logical thing is var "cache-size" in section "state". 
I does getInt() use a different scooping system ?

Reply:
Yes, that's what it means. We discussed it briefly within the team and we decided to put it this way because it's more likely to have other caches configurations in the future than more _states_ tuning.
I'm not quite sure what do you mean by _scooping system_, would you mind to elaborate?